astroquery/mast/tests/test_mast_remote.py

```diff
@@ -519,6 +498,19 @@ def test_observations_download_file_cloud(self, tmp_path, in_uri):
         assert result == ('COMPLETE', None, None)
         assert Path(tmp_path, filename).exists()
 
+    def test_observations_download_file_escaped(self, tmp_path):
+        # test that `download_file` correctly escapes a URI
+        in_uri = 'mast:HLA/url/cgi-bin/fitscut.cgi?' \
+                 'red=hst_04819_65_wfpc2_f814w_pc&blue=hst_04819_65_wfpc2_f555w_pc&size=ALL&format=fits'
+        filename = Path(in_uri).name
+        result = Observations.download_file(uri=in_uri, local_path=tmp_path)
+        assert result == ('COMPLETE', None, None)
+        assert Path(tmp_path, filename).exists()
+
+        # check that downloaded file is a valid FITS file
+        f = fits.open(filename)
```

Comment:
Need to include the path to open the file here. This passes in CI because remote tests aren't run, but I ran it locally.

```suggestion
        f = fits.open(Path(tmp_path, filename))
```